docs/source/guides/user/chapters/concepts.rst

```diff
@@ -8,6 +8,11 @@ It is important to understand some basic concepts before start using Avocado.
 Test Resolution
 ---------------
 
+.. note:: Some definitions here may be out of date. The current runner (run)
+  can still be using some of these definitions in its design, however, we are
+  working on an improved version of the runner, the NextRunner (nrun) that will
```

Comment:
And `nrun` should be gone ASAP, while the NextRunner architecture hopefully will endure... Maybe avoid tying `nrunner` to `nrun`?

Reply:
Yes, I agreed. I will send another version. Was busy today trying to understand the runner and runnable for TAP.